frame/nomination-pools/src/lib.rs

```diff
@@ -1832,17 +1957,19 @@ impl<T: Config> Pallet<T> {
 	/// If the delegator has some rewards, transfer a payout from the reward pool to the delegator.
 	// Emits events and potentially modifies pool state if any arithmetic saturates, but does
 	// not persist any of the mutable inputs to storage.
-	fn do_reward_payout(
+	pub(crate) fn do_reward_payout(
```

Comment:
Why do we increase the visibility?

Reply:
doesn't actually, removed. 26197e383f6349f59da668dbd79b14a39b7890a0